src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,422 @@
+});
+```
+
+This method returns a promise which resolves to an array of predictions ordered by their confidence.
+The number of elements depends on the value you passed to the `alternatives` hint. You could use
+this array to present the user with a choice of possible matches, and have them select an option.
+Alternatively, you can simply go with the most confident prediction, which is what I do in the
+example.
+
+On the prediction object, you will find the recognized text, and an optional segmentation result,
+which I will discuss in the following section.
+
+#### Detailed insights with segmentation results
+
+<!-- 🎉TODO🎉: Image -->
+
+If supported by the target platform, the prediction object can also contain a segmentation result.
+This is an array containing all recognized handwriting segment, a combination of the recognized
+user-identifiable character (`grapheme`) along with its position in the recognized text
+(`beginIndex`, `endIndex`), and the strokes and points that created it. You could use this
+information to track down the recognized graphemes on the canvas again.
+
+```js
+if (mostLikelyPrediction.segmentationResult) {
+  mostLikelyPrediction.segmentationResult.forEach(
+    ({ grapheme, beginIndex, endIndex, drawingSegments }) => {
+      console.log(grapheme, beginIndex, endIndex);
+      drawingSegments.forEach(({ strokeIndex, beginPointIndex, endPointIndex }) => {
+        console.log(strokeIndex, beginPointIndex, endPointIndex);
+      });
+    },
+  );
+}
+```
+
+#### Complete recognition
+
+After the recognition has completed, you can free resources by calling the `clear()` method on the
+`HandwritingDrawing`, and the `finish()` method on the `HandwritingRecognizer`:
+
+```js
+drawing.clear();
+recognizer.finish();
+```
+
+## Demo
+
+The web component `<handwriting-textarea>` implements a
+[progressively enhanced](/progressively-enhance-your-pwa/), editing control capable of handwriting
+recognition. By clicking the button in the lower right corner of the editing control, you activate
+the drawing mode. When you complete the drawing, the web component will automatically start the
+recognition and add the recognized text back to the editing control. If the Handwriting Recognition
+API is not supported at all, or the platform doesn't support the requested features, the edit button
+will be hidden, but the basic editing control remains usable as a `<textarea>`.
+
+The web component offers properties and attributes to define the recognition behavior from the
+outside, including `languages` and `recognitiontype`:
+
+```html
+<handwriting-textarea languages="en,de-DE" recognitiontype="text"></handwriting-textarea>
+```
+
+<!-- 🎉TODO🎉: How to get text in or out? -->
+
+You can try the component using [this demo on Glitch](https://handwriting-recognition.glitch.me/).
+Also be sure to have a look at the
+[source code](https://glitch.com/edit/#!/handwriting-recognition).
+
+<!-- 🎉TODO🎉: Glitch takes ages to start… -->
```

Comment:
I suspect this is because glitch needs to build the demo (because it's not vanilla JS) before serving it.

I had a demo that's 100% vanilla JS, which loads immediately: https://hwr-playground.glitch.me/

Reply:
Added a static build to Glitch now, and point to GitHub for the source code.